src/app/portlets/content-types/fields/content-type-fields-row/content-type-fields-row.component.html

```diff
@@ -0,0 +1,15 @@
+ <div>
+    <span   *ngFor="let column of fieldRow.columns"
```

Comment:
```
<span *ngFor="let column of fieldRow.columns"
    [dragulaModel]='column.fields'
    [style.width]='getSpanWidth()'
    class="fields-container"
    data-drag-type="target"
    dragula='fields-bag'>
```

Maybe like this?

If we do alphabetically attribute order we can have "grouped" bind attr, events and regular attr.